src/main.ts

```diff
@@ -5,7 +5,7 @@ import { AppModule } from './app.module';
 async function startServer(): Promise<void> {
   const PORT = process.env.PORT || 3000;
   const app = await NestFactory.create(AppModule);
-  app.useGlobalPipes(new ValidationPipe());
+  app.useGlobalPipes(new ValidationPipe({ skipMissingProperties: true }));
   await app.listen(3000, () =>
```

Comment:
You are using PORT constant above, and have a hardcoded port 3000 here, which has no sense for me.
Please use PORT variable as 1st argument here.